Flow.Launcher/Languages/en.xaml

```diff
@@ -46,7 +46,7 @@
     <system:String x:Key="browserMorePlugins">Find more plugins</system:String>
     <system:String x:Key="enable">Enable</system:String>
     <system:String x:Key="disable">Disable</system:String>
-    <system:String x:Key="actionKeywords">Action keyword:</system:String>
+    <system:String x:Key="actionKeywords">Query</system:String>
```

Comment:
![image](https://user-images.githubusercontent.com/45326534/125639461-48f7efed-d2d5-47dd-a4ca-b14ceeaa4d39.png)
Is this reference to this? I think Query may be a bit confusing here.